Source/WebKit/glib/profiling.md.in

```diff
@@ -0,0 +1,106 @@
+Title: Profiling
+Slug: profiling
+
+# Profiling with sysprof
+
+## Prerequisites
+
+To work with sysprof one needs to install it first:
+
+- ubuntu:
+```
+apt install sysprof
+```
+- fedora
+```
+dnf install sysprof
+```
+
+## Profiling
+
+To start the profiling one must either:
+
+- start `sysprof-cli` to get a system-wide capture:
+```
+sysprof-cli -f
+```
+- or wrap a command with `sysprof-cli` to get a capture funneled to particular process tree created by the given command:
```

Comment:
Maybe “focused on the execution of a particular process tree” instead of “funneled”?

```suggestion
- or wrap a command with `sysprof-cli` to get a capture focused on the execution of a particular process tree created by the given command:
```